auth/store.go

```diff
@@ -174,17 +175,30 @@ type TokenProvider interface {
 }
 
 type authStore struct {
+	// atomic operations; need 64-bit align, or 32-bit tests will crash
+	revision uint64
+
 	be        backend.Backend
 	enabled   bool
 	enabledMu sync.RWMutex
 
 	rangePermCache map[string]*unifiedRangePermissions // username -> unifiedRangePermissions
 
-	revision uint64
-
 	tokenProvider TokenProvider
 }
 
+func (as *authStore) addRevision() {
```

Comment:
inline this in commitRevision instead of having a function since nothing else increments the revision?